channels/internal/migrations/migrations_cbor_gen.go

```diff
@@ -0,0 +1,876 @@
+// Code generated by github.com/whyrusleeping/cbor-gen. DO NOT EDIT.
```

Comment:
Does the diff of this with the previous internalchannel_cbor_gen.go show no meaningful change?

Reply:
probably? it's all auto generated scripts but the structs are copy paste.